src/VisualStudio/Core/Test/CodeModel/CSharp/ExternalCodePropertyTests.vb

```diff
@@ -0,0 +1,162 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel.CSharp
+    Public Class ExternalCodePropertyTests
+        Inherits AbstractCodePropertyTests
+
+#Region "OverrideKind tests"
+
+        <WorkItem(9646, "https://github.com/dotnet/roslyn/issues/9646")>
+        <ConditionalWpfFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.CodeModel)>
+        Public Async Function TestOverrideKind1() As Task
```

Comment:
I simply copied our existing test coverage from CodePropertyTests. :smile:

I'll add more for both in-source and external properties.